SyncKotlin/app/src/main/java/com/google/android/gms/fit/samples/synckotlin/StepsDatabase.kt

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I would rename this class to MockStepsDatabase.kt, so it's clear it isn't real maybe?